advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx

```diff
@@ -1,24 +1,22 @@
 ---
-title: 'Implementing CipherTrust Transparent Encryption (CTE)'
+title: '3. Implementing CipherTrust Transparent Encryption (CTE)'
 description: 'A walk-through of setting up CTE'
 ---
 
-Implementing the CipherTrust Transparent Encryption (CTE) solution requires the following components:
+**Implementing the CipherTrust Transparent Encryption (CTE) solution requires the following components:**
 
-1. Postgres Server installed and in operation.
+  1. Postgres Server installed and operational.
+  2. CipherTrust Manager installed and operational.
+  3. A CTE agent installed on the Postgres host registered to the CipherTrust Manager.
 
-2. CipherTrust Manager installed and operational.
 
-3. A CTE agent installed on the Postgres host registered to the CipherTrust Manager.
-
-
-The following diagram shows the basic flow of the CTE solution:
+**The following diagram shows the basic flow of the CTE solution:**
 
 <p align="center">
   <img  src="Images/ImplementingCTE.png">
 </p>
 
-### Prerequisites
+### 3.1 Prerequisites
 #### Postgres Host
 1. Ensure that the Postgres Server is installed and running.
```

Comment:
These don't need to be numbered steps. Change Postgres Server to Postgres server.